docs/sites.yml

```diff
@@ -11589,6 +11589,14 @@
     - Business
   built_by: leaniercode
   featured: false
+- title: Fast Golf Scores
```

Comment:
Could you move this to the end of the file as it is a new submission

Reply:
yea sure can do, i just put it here to avoid more conflicts in future when another site is added at bottom